.github/workflows/maven.yml

```diff
@@ -24,5 +24,5 @@ jobs:
     name: Verify
     uses: apache/maven-gh-actions-shared/.github/workflows/maven-verify.yml@v3
     with:
-      ff-maven: "3.8.6"                     # Maven version for fail-fast-build
-      maven-matrix: '[ "3.2.5", "3.8.6" ]'  # Maven versions matrix for verify builds
+      ff-maven: "3.9.2"                     # Maven version for fail-fast-build
```

Comment:
This can be removed

Reply:
When we use matrix for job build, we also need define version for first job ... in this case it will be the same.
But when we upgrade defaults in shared action - we will have different version for first job.